# Lauring Lab at The University of Michigan

Visit the laboratory website [here](<https://lauringlab.wordpress.com/>).

### Repositories by Topic

##### Repositories for Published Work

* [Analysis for McCrone & Lauring, 2016](<https://github.com/lauringlab/Benchmarking_paper>): McCrone, J. T., & Lauring, A. S. (2016). Measurements of Intrahost Viral Diversity Are Extremely Sensitive to Systematic Errors in Variant Calling. Journal of virology, 90(15), 6884–6895. https://doi.org/10.1128/JVI.00667-16

* [Figure Generation for Peck & Lauring, 2018](<https://github.com/lauringlab/JVI_Gem_2018>): Peck, Kayla & Lauring, Adam. (2018). The complexities of viral mutation rates. Journal of virology. 92. 10.1128/JVI.01031-17. 

* [Analysis for Valesano et. al., 2021](<https://github.com/lauringlab/SARS2_Fall_2020>): Valesano, A. L., Fitzsimmons, W. J., Blair, C. N., Woods, R. J., Gilbert, J., Rudnik, D., Mortenson, L., Friedrich, T. C., O'Connor, D. H., MacCannell, D. R., Petrie, J. G., Martin, E. T., & Lauring, A. S. (2021). SARS-CoV-2 Genomic Surveillance Reveals Little Spread From a Large University Campus to the Surrounding Community. Open forum infectious diseases, 8(11), ofab518. https://doi.org/10.1093/ofid/ofab518

* [Analysis for Valesano et. al., 2021](<https://github.com/lauringlab/SARSCov2_Intrahost>): Valesano, A. L., Rumfelt, K. E., Dimcheff, D. E., Blair, C. N., Fitzsimmons, W. J., Petrie, J. G., Martin, E. T., & Lauring, A. S. (2021). Temporal dynamics of SARS-CoV-2 mutation accumulation within and across infected hosts. PLoS pathogens, 17(4), e1009499. https://doi.org/10.1371/journal.ppat.1009499

* [Analysis for Kaul et. al., 2021](<https://github.com/lauringlab/DonorDerivedInfection>): Kaul, D. R., Valesano, A. L., Petrie, J. G., Sagana, R., Lyu, D., Lin, J., Stoneman, E., Smith, L. M., Lephart, P., & Lauring, A. S. (2021). Donor to recipient transmission of SARS-CoV-2 by lung transplantation despite negative donor upper respiratory tract testing. American journal of transplantation : official journal of the American Society of Transplantation and the American Society of Transplant Surgeons, 21(8), 2885–2889. https://doi.org/10.1111/ajt.16532

* [PRE-PRINT Analysis for McCrone et. al., 2020](<https://github.com/lauringlab/IAV_within-host_Ne>): The effective population size and mutation rate of influenza A virus in acutely infected individuals. John T. McCrone, Robert J. Woods, Arnold S. Monto, Emily T. Martin, Adam S. Lauring. bioRxiv 2020.10.24.353748; doi: https://doi.org/10.1101/2020.10.24.353748



##### Pipelines for Data Processing/Analysis

* [Variant Pipeline](<https://github.com/lauringlab/variant_pipeline>)

* [Codon Shuffle Package](<https://github.com/lauringlab/CodonShuffle>): Jorge D M M, Mills R E, Lauring A S, CodonShuffle: a tool for generating and analyzing synonymously mutated sequences. Virus Evolution, 2015, 1(1): vev012

##### General Use

* [Using Slack for Alerts & Updates](<https://github.com/lauringlab/AlertCode>)

##### RShiny Applications

* [Displaying Zip Code and County Level SARS-CoV-2 Data](<https://github.com/lauringlab/sapphire_covid_display>)

* [Processing SARS-CoV-2 Data Post-Testing](<https://github.com/lauringlab/sarscov2_application>)

* [Validation Analysis for McCrone & Lauring, 2016](<https://github.com/lauringlab/benchmarking_shiny>): McCrone, J. T., & Lauring, A. S. (2016). Measurements of Intrahost Viral Diversity Are Extremely Sensitive to Systematic Errors in Variant Calling. Journal of virology, 90(15), 6884–6895. https://doi.org/10.1128/JVI.00667-16

* [Exploring iSNV, Particularly Tranmission Pairs](<https://github.com/lauringlab/Host_level_IAV_app>)

##### Other


##### No Longer Maintained
