![](https://github.com/lauringlab/.github/blob/main/Michigan-Medicine_Logo-Horizontal-WEBHEX.png)
![](https://github.com/lauringlab/.github/blob/main/U-M_Logo-Horizontal-Hex.png)

# Lauring Lab at The University of Michigan

Visit the laboratory website [here](<https://lauringlab.wordpress.com/>).

### Repositories by Topic

##### Repositories for Published Work

* [Analysis for Valesano et. al., 2021](<https://github.com/lauringlab/SARS2_Fall_2020>): Valesano, A. L., Fitzsimmons, W. J., Blair, C. N., Woods, R. J., Gilbert, J., Rudnik, D., Mortenson, L., Friedrich, T. C., O'Connor, D. H., MacCannell, D. R., Petrie, J. G., Martin, E. T., & Lauring, A. S. (2021). SARS-CoV-2 Genomic Surveillance Reveals Little Spread From a Large University Campus to the Surrounding Community. Open forum infectious diseases, 8(11), ofab518. https://doi.org/10.1093/ofid/ofab518

* [Analysis for Valesano et. al., 2021](<https://github.com/lauringlab/SARSCov2_Intrahost>): Valesano, A. L., Rumfelt, K. E., Dimcheff, D. E., Blair, C. N., Fitzsimmons, W. J., Petrie, J. G., Martin, E. T., & Lauring, A. S. (2021). Temporal dynamics of SARS-CoV-2 mutation accumulation within and across infected hosts. PLoS pathogens, 17(4), e1009499. https://doi.org/10.1371/journal.ppat.1009499

* [Analysis for Kaul et. al., 2021](<https://github.com/lauringlab/DonorDerivedInfection>): Kaul, D. R., Valesano, A. L., Petrie, J. G., Sagana, R., Lyu, D., Lin, J., Stoneman, E., Smith, L. M., Lephart, P., & Lauring, A. S. (2021). Donor to recipient transmission of SARS-CoV-2 by lung transplantation despite negative donor upper respiratory tract testing. American journal of transplantation : official journal of the American Society of Transplantation and the American Society of Transplant Surgeons, 21(8), 2885–2889. https://doi.org/10.1111/ajt.16532

* [Analysis for Baang et. al., 2021](<https://github.com/lauringlab/ProlongedReplicationCase>): Baang, J. H., Smith, C., Mirabelli, C., Valesano, A. L., Manthei, D. M., Bachman, M. A., Wobus, C. E., Adams, M., Washer, L., Martin, E. T., & Lauring, A. S. (2021). Prolonged Severe Acute Respiratory Syndrome Coronavirus 2 Replication in an Immunocompromised Patient. The Journal of infectious diseases, 223(1), 23–27. https://doi.org/10.1093/infdis/jiaa666

* [Analysis for Valesano et. al., 2021](<https://github.com/lauringlab/Poliovirus_Intrahost>): Valesano, A. L., Taniuchi, M., Fitzsimmons, W. J., Islam, M. O., Ahmed, T., Zaman, K., Haque, R., Wong, W., Famulare, M., & Lauring, A. S. (2021). The Early Evolution of Oral Poliovirus Vaccine Is Shaped by Strong Positive Selection and Tight Transmission Bottlenecks. Cell host & microbe, 29(1), 32–43.e4. https://doi.org/10.1016/j.chom.2020.10.011

* [PRE-PRINT Analysis for McCrone et. al., 2020](<https://github.com/lauringlab/IAV_within-host_Ne>): The effective population size and mutation rate of influenza A virus in acutely infected individuals. John T. McCrone, Robert J. Woods, Arnold S. Monto, Emily T. Martin, Adam S. Lauring. bioRxiv 2020.10.24.353748; doi: https://doi.org/10.1101/2020.10.24.353748

* [Analysis for Valesano et. al., 2020](<https://github.com/lauringlab/Host_level_IBV_evolution>): Valesano, A. L., Fitzsimmons, W. J., McCrone, J. T., Petrie, J. G., Monto, A. S., Martin, E. T., & Lauring, A. S. (2020). Influenza B Viruses Exhibit Lower Within-Host Diversity than Influenza A Viruses in Human Hosts. Journal of virology, 94(5), e01710-19. https://doi.org/10.1128/JVI.01710-19

* [Analysis for McCrone et. al., 2018](<https://github.com/lauringlab/Host_level_IAV_evolution>): McCrone, J. T., Woods, R. J., Martin, E. T., Malosh, R. E., Monto, A. S., & Lauring, A. S. (2018). Stochastic processes constrain the within and between host evolution of influenza virus. eLife, 7, e35962. https://doi.org/10.7554/eLife.35962

* [Analysis for Fitzsimmons et. al., 2018](<https://github.com/lauringlab/speed_fidelity>): Fitzsimmons, W. J., Woods, R. J., McCrone, J. T., Woodman, A., Arnold, J. J., Yennawar, M., Evans, R., Cameron, C. E., & Lauring, A. S. (2018). A speed-fidelity trade-off determines the mutation rate and virulence of an RNA virus. PLoS biology, 16(6), e2006459. https://doi.org/10.1371/journal.pbio.2006459

* [Figure Generation for Peck & Lauring, 2018](<https://github.com/lauringlab/JVI_Gem_2018>): Peck, Kayla & Lauring, Adam. (2018). The complexities of viral mutation rates. Journal of virology. 92. 10.1128/JVI.01031-17. 

* [Analysis for Lyons et. al, 2017](<https://github.com/lauringlab/TsTv_paper>): Lyons, D. M., & Lauring, A. S. (2017). Evidence for the Selective Basis of Transition-to-Transversion Substitution Bias in Two RNA Viruses. Molecular biology and evolution, 34(12), 3205–3215. https://doi.org/10.1093/molbev/msx251

* [Analysis for Debbink et. al., 2017](<https://github.com/lauringlab/Fluvacs_paper>): Debbink, K., McCrone, J. T., Petrie, J. G., Truscon, R., Johnson, E., Mantlo, E. K., Monto, A. S., & Lauring, A. S. (2017). Vaccination has minimal impact on the intrahost diversity of H3N2 influenza viruses. PLoS pathogens, 13(1), e1006194. https://doi.org/10.1371/journal.ppat.1006194

* [Analysis for Visher et. al., 2016](<https://github.com/lauringlab/MFE_paper>): Visher, E., Whitefield, S. E., McCrone, J. T., Fitzsimmons, W., & Lauring, A. S. (2016). The Mutational Robustness of Influenza A Virus. PLoS pathogens, 12(8), e1005856. https://doi.org/10.1371/journal.ppat.1005856

* [Analysis for McCrone & Lauring, 2016](<https://github.com/lauringlab/Benchmarking_paper>): McCrone, J. T., & Lauring, A. S. (2016). Measurements of Intrahost Viral Diversity Are Extremely Sensitive to Systematic Errors in Variant Calling. Journal of virology, 90(15), 6884–6895. https://doi.org/10.1128/JVI.00667-16


##### Pipelines for Data Processing/Analysis

* [Variant Pipeline](<https://github.com/lauringlab/variant_pipeline>)

* [Codon Shuffle Package](<https://github.com/lauringlab/CodonShuffle>): Jorge D M M, Mills R E, Lauring A S, CodonShuffle: a tool for generating and analyzing synonymously mutated sequences. Virus Evolution, 2015, 1(1): vev012


##### General Use

* [Using Slack for Alerts & Updates](<https://github.com/lauringlab/AlertCode>)


##### RShiny Applications

* [Displaying Zip Code and County Level SARS-CoV-2 Data](<https://github.com/lauringlab/sapphire_covid_display>)

* [Processing SARS-CoV-2 Data Post-Testing](<https://github.com/lauringlab/sarscov2_application>)

* [Validation Analysis for McCrone & Lauring, 2016](<https://github.com/lauringlab/benchmarking_shiny>): McCrone, J. T., & Lauring, A. S. (2016). Measurements of Intrahost Viral Diversity Are Extremely Sensitive to Systematic Errors in Variant Calling. Journal of virology, 90(15), 6884–6895. https://doi.org/10.1128/JVI.00667-16

* [Exploring iSNV, Particularly Tranmission Pairs](<https://github.com/lauringlab/Host_level_IAV_app>)


##### Other

* [Assay Analysis - Illumina, Influenza A](<https://github.com/lauringlab/NGS_mutation_rate_assay>); Assay used in Pauly, M. D., Procario, M. C., & Lauring, A. S. (2017). A novel twelve class fluctuation test reveals higher than expected mutation rates for influenza A viruses. eLife, 6, e26437. https://doi.org/10.7554/eLife.26437


